ServerlessLibraryUI/src/components/AppTopbar.vue

```diff
@@ -28,6 +28,7 @@
         <select class="c-search dropdown2"  v-model="language">
           <option value="" selected >Language: All</option>
```

Comment:
so the order would be: JavaScript, C#, Python, Java
@ColbyTresness could you ACK this?

Reply:
I think this is good for now. If/when we add more languages we can reorder/alphabetize.